core/src/utils/directive.ts

```diff
@@ -235,7 +235,12 @@ export interface AttributesDirectiveProps {
 	 * @remarks
 	 * Key-value pairs where keys are event types and values are event handlers.
 	 */
-	events?: Partial<{[K in keyof HTMLElementEventMap]: (this: HTMLElement, event: HTMLElementEventMap[K]) => void}>;
+	events?: Partial<{
+		[K in keyof HTMLElementEventMap]: {
+			eventFn: (this: HTMLElement, event: HTMLElementEventMap[K]) => void;
```

Comment:
```suggestion
			handler: (this: HTMLElement, event: HTMLElementEventMap[K]) => void;
```
I suggest to name it `handler` rather than `eventFn`.